core-web/libs/sdk/analytics/README.md

```diff
@@ -0,0 +1,115 @@
+# @dotcms/analytics
+
+`@dotcms/analytics` is the official dotCMS JavaScript library that helps track events and analytics in your webapps. Currently available as an IIFE (Immediately Invoked Function Expression) module for direct browser usage, with planned future support for React and Next.js applications.
+
+## Features
+
+-   **Simple Browser Integration**: Easy to implement via script tags using IIFE implementation
+-   **Event Tracking**: Simple API to track custom events with additional properties
+-   **Automatic PageView**: Option to automatically track page views
+-   **Debug Mode**: Optional debug logging for development
+
+## Installation
+
+Include the script in your HTML page:
+
+```html
+<script src="analytics.iife.js"></script>
+```
+
+## Configuration
+
+The script can be configured using data attributes:
+
+-   **data-analytics-server**: URL of the server where events will be sent. If not provided, it defaults to the current location (window.location.href)
+-   **data-analytics-debug**: Presence of this attribute enables debug logging (no value needed)
+-   **data-analytics-auto-page-view**: Presence of this attribute enables automatic page view tracking (no value needed)
+-   **data-analytics-key**: API key for authentication with the analytics server
+
+Examples:
+
+```html
+<!-- Basic usage with defaults -->
+<script src="ca.min.js"></script>
+
+<!-- With custom server and API key -->
+<script
+    src="ca.min.js"
+    data-analytics-server="http://your-server:8080"
+    data-analytics-key="your-api-key"></script>
+
+<!-- With all options -->
+<script
+    src="ca.min.js"
+    data-analytics-server="http://your-server:8080"
+    data-analytics-key="your-api-key"
+    data-analytics-debug
+    data-analytics-auto-page-view></script>
+```
```

Comment:
Do I need another server to run analytics?
How do I get the api key?

Reply:
- All works with the infrastructure of Analytics.
- The key comes from the DotCMS Analytics app